images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -35,5 +35,13 @@ Choco-Install -PackageName cabal
 Invoke-PesterTests -TestFile 'Haskell'
 
 # install minimal ghcup, utilizing pre-installed msys2 at C:\msys64
-$bootstrapHaskell = Invoke-WebRequest https://www.haskell.org/ghcup/sh/bootstrap-haskell.ps1 -UseBasicParsing
-Invoke-Command -ScriptBlock ([ScriptBlock]::Create($bootstrapHaskell)) -ArgumentList $false, $true, $true, $false, $false, $false, $false, C:\, "", C:\msys64, C:\cabal
+$msysPath = "C:\msys64"
+$ghcupPrefix = "C:\"
+$appdata = [Environment]::GetEnvironmentVariable('APPDATA', [System.EnvironmentVariableTarget]::Machine) 
```

Comment:
AppData env var does not exist in Machine scope.

Reply:
At the end of image generation process packer invokes sysprep util with /generalize param and removes the user which it uses during generation:
`The Sysprep /generalize command removes unique information from a Windows installation so that you can reuse that image on different computers.`

The value of %AppData% is C:\Users\\**USERNAME**\\AppData\Roaming <- That's why it exists only in the USER scope.

We should use the hardcoded variant as before, for example: **$cabalDir = "C:\cabal"**

Reply:
@hasufell, Could you please apply changes above? ^^^

Reply:
@al-cheb sure, bet that won't fix the issue with ghcup not being in PATH